### ISSUE TITLE
Ligature request: Swift \(

### ISSUE THREAD

**Issue by pxeger** [tonsky/FiraCode#1179]
In Swift, `\(` is used for string interpolation:

```swift
var name = "Tim McGraw"
"Your name is \(name)"
```

([source](https://www.hackingwithswift.com/read/0/5/string-interpolation))

Could a ligature be added for this?

**Comment by tonsky** [comment]
Sure! Do you have ideas how it might look like?

**Comment by pxeger** [comment]
The backslash and the opening parenthesis could be squished together so that the end of the slash merges with the tip of the parenthesis, something like:

![image](https://user-images.githubusercontent.com/67427173/103368894-aa85be00-4ac0-11eb-8cf1-f7747c904a77.png)

**Comment by alexeyten** [comment]
This would conflict with regexps

**Comment by RubenVerg** [comment]
As I have frequently said, a "escape for literal open bracket" is a single character in my mind, so no conflict imho

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by jwortmann** [comment]
In LaTeX, `\( ... \)` is used for the inline math environment. Hence, if `\(` gets added, I would expect a similar ligature for `\)` as well, so that it looks symmetric.

**Comment by ThatXliner** [comment]
Agreed. Heck, why not `\[` and `\]` have their own ligatures, too? And a whole set of ligatures for regex e.g. `(?=`, etc (I suck at memorizing regex lookbehind/ahead so correct me if I'm wrong :sweat_smile:).

**Comment by CodeCrazy-ywt** [comment]
> This would conflict with regexps

Would it be better if these two symbols just get closer but not connected?
